src/ssh/azext_ssh/custom.py

```diff
@@ -50,12 +51,30 @@ def _do_ssh_op(cmd, resource_group, vm_name, ssh_ip, public_key_file, private_ke
 
 
 def _get_and_write_certificate(cmd, public_key_file, cert_file):
-    scopes = ["https://pas.windows.net/CheckMyAccess/Linux/.default"]
+    cloudtoscope = {
+        "azurecloud": "https://pas.windows.net/CheckMyAccess/Linux/.default",
+        "azurechinacloud": "https://pas.chinacloudapi.cn/CheckMyAccess/Linux",
```

Comment:
I think you forgot the /.default at the end

Reply:
I tested this change, withouth the /.default at the end it does not work, with it I seem to get further along in the auth process :)